charts/vega-metrics-agent/values.yaml

```
@@ -12,7 +12,7 @@ serviceAccount:
   name: vega-metrics-sa
 image:
   repository: public.ecr.aws/c0f8b9o4/vegacloud/vega-metrics-agent
-  tag: 1.1.0
+  tag: 1.1.1
```

```suggestion
  tag: 1.1.2
```
per spencer's comment